site/content/integrate/apps/quickstart/_index.md

```diff
@@ -28,6 +28,8 @@ You can also access Mattermost from outside the Docker network via [http://local
 
 To change the Mattermost Apps plugin or the Mattermost Server versions, you can edit [`docker-compose.yml`](https://github.com/mattermost/mattermost-app-examples/blob/master/docker-compose.yml).
 
+Be sure to create a `.docker.env` file in the same directory as the `docker-compose.yml` file to store any secrets. Or you can rename and use [`.docker.env.example`](https://github.com/mattermost/mattermost-app-examples/blob/master/.docker.env.example) in the repository to see some example secrets you may use while building Apps.
```

Comment:
This is worded in a way that makes me think "in case you need to store secrets, you'll need to create this file", when in reality, it's required to make this run. Feel free to reword parts of the suggestion below:

```suggestion
You'll need to create a `.docker.env` file in the same directory as the `docker-compose.yml` file in order to complete the Docker Compose configuration. Use this file to store any secrets or any other environment variables you wish to use with the Mattermost server. Check out the [`.docker.env.example`](https://github.com/mattermost/mattermost-app-examples/blob/master/.docker.env.example) file in the examples repository to see some secrets you may want to use while building Apps. The `.docker.env` file is ignored in the repository's `.gitignore` file, so these secrets won't be committed to the repository if you make any edits.
```

Reply:
Agreed! Great suggestion.